app/models/attribute.py

```diff
@@ -0,0 +1,27 @@
+from .. import db
+
+users_association_table = db.Table(
+    'users_association', db.Model.metadata,
+    db.Column('tags_id', db.Integer, db.ForeignKey('tags.id')),
```

Comment:
I think for consistency with other foreign keys we should call this `db.Column('tag_id', ...)`